layouts/partials/data_jobs/setup-platforms.html

```diff
@@ -1,50 +1,68 @@
 {{ $dot := . }}
 <div class="dsm-containers">
   <div class="container cards-dd">
-    <div class="row row-cols-1 row-cols-md-5 g-2 g-xl-3 justify-content-sm-center">
-
-      <div class="col">
-        <a class="card h-100" href="/data_jobs/emr">
-          <div class="card-body text-center py-2 px-1">
-            {{ partial "img.html" (dict "root" . "src" "integrations_logos/amazon_emr.png" "class" "img-fluid" "alt"
-            "Amazon EMR" "width" "200") }}
-          </div>
-        </a>
-      </div>
-
-      <div class="col">
-        <a class="card h-100" href="/data_jobs/databricks/">
-          <div class="card-body text-center py-2 px-1">
-            {{ partial "img.html" (dict "root" . "src" "integrations_logos/databricks.png" "class" "img-fluid" "alt" "Databricks" "width" "200") }}
-          </div>
-        </a>
-      </div>
-
-      <div class="col">
-        <a class="card h-100" href="/data_jobs/dataproc/">
-          <div class="card-body text-center py-2 px-1">
-            {{ partial "img.html" (dict "root" . "src" "integrations_logos/google_cloud_dataproc.png" "class"
-            "img-fluid" "alt" "GCP Dataproc" "width" "200") }}
-          </div>
-        </a>
-      </div>
-
-      <div class="col">
-        <a class="card h-100" href="/data_jobs/kubernetes/">
-          <div class="card-body text-center py-2 px-1">
-            {{ partial "img.html" (dict "root" . "src" "integrations_logos/kubernetes.png" "class" "img-fluid" "alt"
-            "Kubernetes" "width" "200") }}
-          </div>
-        </a>
-      </div>
-
-      <div class="col">
-        <a class="card h-100" href="/data_jobs/airflow/">
-          <div class="card-body text-center py-2 px-1">
-            {{ partial "img.html" (dict "root" . "src" "integrations_logos/airflow.png" "class" "img-fluid" "alt" "Airflow" "width" "200") }}
-          </div>
-        </a>
-      </div>
+    <div class="row row-cols-1 row-cols-md-4 g-2 g-xl-3 justify-content-sm-center">
+
+     <div class="col">
+      <a class="card h-100" href="/data_jobs/databricks/">
+        <div class="card-body text-center py-2 px-1">
+          {{ partial "img.html" (dict "root" . "src" "integrations_logos/databricks.png" "class" "img-fluid" "alt" "Databricks" "width" "200") }}
+        </div>
+      </a>
+     </div>
+
+     <div class="col">
+      <a class="card h-100" href="/data_jobs/airflow/">
+        <div class="card-body text-center py-2 px-1">
+          {{ partial "img.html" (dict "root" . "src" "integrations_logos/airflow.png" "class" "img-fluid" "alt" "Airflow" "width" "200") }}
+        </div>
+      </a>
+     </div>
+
+     <div class="col">
+      <a class="card h-100" href="/data_observability/jobs_monitoring/dbtcore">
```

Comment:
I see we don't have 2 different logos for dbt core and dbt cloud which is confusing. Can we combine this into one dbt page instead of 2 with different tabs for setup steps per platforms (like we have for Airflow)? So it would be a "dbt core" setup tab and one for "dbt cloud" The overview/next sections are similar between the two anyway anyway.

<img width="428" height="196" alt="Screenshot 2025-12-24 at 1 14 09 PM" src="https://github.com/user-attachments/assets/de3cbf70-f39c-43c4-97a3-d4e5a2681044" />


<img width="758" height="429" alt="Screenshot 2025-12-24 at 1 07 15 PM" src="https://github.com/user-attachments/assets/1ad2346f-23c0-4182-964e-b2bde2733271" />